app/operations/assignments/create.rb

```diff
@@ -22,7 +22,7 @@ def execute
         subjects: []
       }
 
-      panoptes.add_subjects_to_subject_set(subject_set["id"], subject_ids)
+      FillSubjectSetWorker.perform_async(subject_set["id"], subject_ids)
```

Comment:
It's a Sidekiq thing: https://github.com/mperham/sidekiq/wiki/Getting-Started

Doesn't work everywhere in Ruby, sadly. Although with a Sidekiq alternative called DelayedJob, you can actually do `whatever.delay.some_method(a, b, c)` and it'll magically work on anything. That magic bites you in the ass at some point though.